docs/release-notes/release-notes-0.6.3.md

```diff
@@ -0,0 +1,221 @@
+Joinmarket-clientserver 0.6.3:
+=================
+
+<https://github.com/joinmarket-org/joinmarket-clientserver/releases/tag/v0.6.3>
+
+**Note that this will be the last release supporting Python 3.5 or lower.**
+
+Includes a few notable improvements to Qt as well as a few quite important back-end
+bugfixes. Upgrading is recommended as soon as possible.
+
+Please report bugs using the issue tracker at github:
+
+<https://github.com/joinmarket-org/joinmarket-clientserver/issues>
+
+Upgrading 
+=========
+
+To upgrade:
+
+If you are upgrading from a version pre-0.6.2 then please see the section "Move user data to home directory" in 
+[the 0.6.2 release notes](https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/master/docs/release-notes/release-notes-0.6.2.md),
+and follow the instructions there (except the `commitmentlist` file - that can actually
+be left alone, the previous release notes were wrong on this point).
+
+First run the `install.sh` script as mentioned in the README. When prompted to overwrite the directory `jmvenv`, accept.
+
+**Reminder: Python 3 is required; Python 2 is no longer supported as it reached EOL.**
+
+Note that `./install.sh -?` will show the options for installation.
+
+Notable changes
+===============
+
+### Lock file name change
+
+While this change is very minor, it can cause confusion if un-noticed. Previously the wallet was kept single access with a file, named wallet.jmdat.lock (or \*.lock where \* was whatever your wallet name is). Since this could result in accidental deletion of the wallet via tab completion, it was decided to change this to .\*.lock - note that many file system explorers/commands will hide this file by default as it's considered of a 'hidden' type. As a reminder, this file must be deleted in cases where there was a system or program crash (you are reminded of this in case it happens).
```

Comment:
`as it's considered of a 'hidden' type.`
Delete *of* ?

Reply:
i think "of the type X" is pretty normal English, no? But without "of" here would work too, sure.